trymito.io/components/InstallInstructions/InstallInstructions.tsx

```diff
@@ -15,16 +16,17 @@ const InstallInstructions = (props: {}): JSX.Element => {
                 in under a minute
```

Comment:
Maybe we should say "In two lines of code" instead of "in under a minute" because we know on some user's computers this could take 45 minutes to run if they need to install everything.